src/ports/linux/pnal.c

```diff
@@ -307,11 +307,17 @@ static pnal_eth_mau_t calculate_mau_type (
          switch (speed)
          {
          case SPEED_10:
-            return PNAL_ETH_MAU_COPPER_10BaseT;
+            return PNAL_ETH_MAU_COPPER_10BaseTX_FULL_DUPLEX;
```

Comment:
Had a brief review with @wkz and this was the only thing he got a knee-jerk reaction to.  Turns out I added this misnomer in 166f71df.  Would be great if you could add a commit to this branch that corrects my change.  

I.e., rename all `*10BaseTX*` to `*10BaseT*` since -TX only arrived with 100 Mbps.  (ref: https://en.wikipedia.org/wiki/Ethernet_over_twisted_pair)